Sources/Shared/Data Manager/Proxies/Social/SocialPostActivity.m

```diff
@@ -98,11 +98,16 @@ -(void)postActivity:(NSString *)message fileURL:(NSString*)fileURL fileName:(NSS
         ApplicationPreferencesManager *serverPM = [ApplicationPreferencesManager sharedInstance];
         activity.type = @"DOC_ACTIVITY"; 
         
+        NSString* docPath;
+        if (space){
+            docPath = [NSString stringWithFormat:@"/Groups%@/Documents/Mobile/%@",space.groupId,fileName];
+        } else {
+            docPath = [NSString stringWithFormat:@"%@/Public/Mobile/%@",serverPM.userHomeJcrPath,fileName];
```

Comment:
You can use `MOBILE_UPLOAD_DEST_FOLDER` instead of `Mobile` in the 2 paths above.